### PR TITLE
PrivCash

### PR DESCRIPTION
**Pull request by chervyachok**
##### Name (to be shown on DefiLlama):
PrivCash

##### Twitter Link:
https://twitter.com/privcash_

##### List of audit links if any:
N/A

##### Website Link:
https://priv.cash

##### Logo (High resolution, will be shown with rounded borders):
https://priv.cash/img/privcash_hd_logo.png

##### Current TVL:
N/A

##### Treasury Addresses (if the protocol has treasury)
N/A

##### Chain:
KAVA

##### Coingecko ID (so your TVL can appear on Coingecko, leave empty if not listed): (https://api.coingecko.com/api/v3/coins/list)
N/A

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap, leave empty if not listed): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
N/A

##### Short Description (to be shown on DefiLlama):
PrivCash is a decentralized protocol for private transactions on the Kava network

##### Token address and ticker if any:
N/A

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Privacy

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
N/A

##### forkedFrom (Does your project originate from another project):
https://github.com/tornadocash/tornado-core

##### methodology (what is being counted as tvl, how is tvl being calculated):
All deposit nominal contracts (KAVA + USDC + PRIV)

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/privcash exports TVL: 
        
 
 
    kava                      0
    
    total                    0

File: projects/privcash/config.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      holders: [
+        "0xe4e992802314dbbd8BB9d050afae19ca1c45cB1A", // 10 
+        "0x8DFB4d1925cC8C7446AfA92f1cDd6c8be567Ae7C", // 100 
+        "0x00F5E31F0E33FBc23e723dCEd6C078fdD688D36a", // 1000 
+        "0xCA0d7b385e9DC484C646C50F1BBA6B01CC60E361", // 10000 
+        "0xc500DA72cCeA705aD5Ee3A4d77ABb1864DD30a4F", // 100000 
+      ],
+    },
+    {
+      tokens: ['0x5Ba0Ab8f8E04419b91B0387d1A86FdaE7904B5e6'],   // PRIV
```
Comment (g1nt0ki) [inline projects/privcash/config.js:26]: sorry, you cant count your own tokens as tvl. We will wait till there is some tvl before listing.

**Comment by llamatester** [comment]
The adapter at projects/privcash exports TVL: 
        
 
 
    kava                      10.009999999999998
    
    total                    10.009999999999998

**Comment by g1nt0ki** [comment]
@chervyachok thanks for fixing it

**Pull request merged**